settings.gradle

```diff
@@ -38,4 +38,6 @@ include 'data-prepper-plugins:blocking-buffer'
 include 'data-prepper-plugins:http-source'
 include 'data-prepper-plugins:grok-prepper'
 include 'data-prepper-logstash-configuration'
+include 'e2e'
```

Comment:
I want to raise the question of what name to use here. This uses `e2e`, but there may be some other options: `e2e-test`, `qa`, `qa-test`, `test`.

I don't like the name `test` since it is ambiguous with the other test tasks. I do think I like the idea of a clearer name like `e2e-test` or `qa-test` since `e2e` may not be an immediately obvious convention.

Also, the core OpenSearch project uses `qa` for its end-to-end tests.

Reply:
@dlvenable I also went back and forth between qa and e2e. I think qa-test looks good to me.

Reply:
@cmanning09 @graytaylor0 , Any thoughts on the name?

Reply:
Actually e2e-test might be better since e2e is already populated in the repo 😄